tinacms/tinacms
# Feat/naviate single doc collections

## What this does.

Video: https://www.loom.com/share/854b53db99bb406ab5bd8627224f6d53

If you have a collection with 

```ts
ui: {
  allowedActions: {
    create: false,
    delete: false,
  }
},
```

and it only contains one document. Instead of navigating to the collection list page it will navigate to the document edit page.


## How to test

Add 


```ts
ui: {
  allowedActions: {
    create: false,
    delete: false,
  }
},
```
To a collection that only contains one document, click on that collection in that admin view, it should navigate directly to that document.
